docs/en/observability/metrics-threshold-alert.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[metrics-threshold-alert]]
+= Create a metrics threshold alert
+
+Based on the metrics that are listed on the *Metrics Explorer* page within the {metrics-app},
+you can create a threshold alert to notify you when a metric has reached or exceeded a value for a specific
+time period.
+
+Additionally, each alert can be defined using multiple
+conditions that combine metrics and thresholds to create precise notifications.
+
+. In the side navigation, expand *Observability*, click *Metrics*, and then click *Metrics Explorer*.
+. On the *Metrics Explorer* page, click *Alerts*, and then select *Create alert*.
+
+[[metrics-conditions]]
+== Metric conditions
+
+Conditions for each alert can be applied to specific metrics that you select. You can select the aggregation type,
+the metric, and specify the threshold value to be alerted on. To help you determine which thresholds are meaningful
+to you, the preview charts provide a visualization. You can also select to get alerted when the metric does not
+report any data over the expected time period, or if the alert fails to query {es}. 
+
```

Comment:
Describe what the following options mean:

- Alert me if there's no data - fires an alert if the metric(s) don't report any data over the expected time period or if the alert fails to query Elasticsearch.
- Filters:  controls the scope of an alert
- Create alert per: creates an alert instance for every unique value of the `field` added here. Examples: Create an alert per every host, Create an alert per every mount point of every host. Multiple fields can be added.

Reply:
Added to the docs. Thanks.